From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dominik Csapak <d.csapak@proxmox.com>
Date: Tue, 16 May 2017 10:36:15 +0200
Subject: [PATCH] use only app.js

As PVE now uses the commonjs version, there's only need to fetch the
generated app.js

Signed-off-by: Dominik Csapak <d.csapak@proxmox.com>
---
 vnc.html | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/vnc.html b/vnc.html
index 5f09cd2..9f594e2 100644
--- a/vnc.html
+++ b/vnc.html
@@ -39,12 +39,7 @@
     </script>
 
     <!-- begin scripts -->
-    <!-- promise polyfills promises for IE11 -->
-    <script src="/novnc/vendor/promise.js"></script>
-    <!-- ES2015/ES6 modules polyfill -->
-    <script nomodule src="/novnc/vendor/browser-es-module-loader/dist/browser-es-module-loader.js"></script>
-    <!-- actual script modules -->
-    <script type="module" crossorigin="anonymous" src="/novnc/app/ui.js"></script>
+    <script crossorigin=anonymous" src="/novnc/app.js"></script>
     <!-- end scripts -->
 </head>
 
